src/pandablocks_ioc/_pvi.py

```diff
@@ -375,4 +378,4 @@ def create_pvi_records(record_prefix: str):
 
                 Pvi.add_general_device_refs_to_groups(device)
 
-                formatter.format(device, record_prefix + ":", bobfile_path)
+                formatter.format(device, bobfile_path)
```

Comment:
The formatter has changed so now it doesn't take a prefix - it expects all the read/write pv's in the signals to have the prefixes already. We have to add this to signals above.